Packs/SailPointIdentityNow/Integrations/SailPointIdentityNow/SailPointIdentityNow.py

```diff
@@ -0,0 +1,622 @@
+    """
+    Create header with OAuth 2.0 authentication information.
+
+    :type base_url: ``str``
+    :param base_url: Base URL of the IdentityNow tenant.
+
+    :type client_id: ``str``
+    :param client_id: Client Id for OAuth 2.0.
+
+    :type client_secret: ``str``
+    :param client_secret: Client Secret for OAuth 2.0.
+
+    :type grant_type: ``str``
+    :param grant_type: Grant Type for OAuth 2.0. Defaulted to 'client_credentials' if not provided.
+
+    :return: Header with OAuth 2.0 information if client_id & client_secret are provided, else None.
+    This will return None if the client_id & client_secret were not valid (authorized).
+    """
+    if base_url is None or client_id is None or client_secret is None:
+        return None
+
+    if grant_type is None:
+        grant_type = 'client_credentials'
+
+    params = {
+        'grant_type': grant_type,
+        'client_id': client_id,
+        'client_secret': client_secret
+    }
+    oauth_response = requests.request("POST", url=f'{base_url}{IDN_OAUTH_EXT}', params=params)
+    if oauth_response is not None and 200 <= oauth_response.status_code < 300:
+        return {
+            'Authorization': 'Bearer %s' % oauth_response.json().get('access_token', None),
+            'Content-Type': 'application/json'
+        }
+    else:
+        return None
+
+
+def build_query_object(object_type: str, query: str):
+    """
+    Build query object for IdentityNow Search Engine.
+
+    :type object_type: ``str``
+    :param object_type: Type of object (Identity, AccessProfile, Account, Role, Entitlement, Event)
+
+    :type query: ``str``
+    :param query: Elastic search query for the IdentityNow Search Engine.
+
+    :return: JSON object that contains the elastic search query.
+    """
+    if object_type is None or object_type not in SEARCHABLE_TYPE_LIST:
+        return None
+
+    if query is None:
+        return None
+
+    return {
+        "indices": [object_type],
+        "query": {
+            "query": query
+        }
+    }
+
+
+def get_markdown(object_type: str, objects=None):
+    """
+    Getting markdown for object type to display the results in human readable format.
+
+    :type object_type: ``str``
+    :param object_type: Type of IdentityNow object.
+
+    :type objects: ``dict`` or ``list``
+    :param objects: Single or list of Identity resources object/s.
+
+    :return: Markdown for each object type.
+    """
+    markdown = ''
+    if object_type == 'IdentityNow.Identity':
+        headers = ['id', 'name', 'displayName', 'firstName', 'lastName', 'email', 'created', 'modified', 'inactive',
+                   'protected', 'status', 'isManager', 'identityProfile', 'source', 'attributes', 'accounts',
+                   'accountCount', 'appCount', 'accessCount', 'entitlementCount', 'roleCount', 'accessProfileCount',
+                   'pod', 'org', 'type']
+        markdown = tableToMarkdown('Identity(Identities)', objects, headers=headers)
```

Comment:
```suggestion
        markdown = tableToMarkdown('Identity(Identities)', objects, headers=headers, removeNull=True)
```

This will remove the null values from the table, suggested for all tableToMrkdown uses

Reply:
This is added to other calls in the function as well.